tests/framework/validators/UniqueValidatorTest.php

```diff
@@ -434,4 +434,30 @@ public function testExpresionInAttributeColumnName()
         $validator->validateAttribute($model, 'title');
         $this->assertFalse($model->hasErrors(), 'There were errors: ' . json_encode($model->getErrors()));
     }
+
+    /**
+     * Test validating a class with default scope
+     * @see https://github.com/yiisoft/yii2/issues/14484
+    */
+    public function testFindModelWith()
+    {
+        $validator = new UniqueValidator([
+            'targetAttribute' => ['status', 'profile_id']
+        ]);
+
+       
+        $model = WithCustomer::find()->one();
+
+        $validator->validateAttribute($model, 'dummy');
```

Comment:
don't we need any assertion + can it be made more descriptive for an easier understanding?

Reply:
@dynasource please, review again